plugin/storage/cassandra/spanstore/dbmodel/tag_filter.go

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"github.com/uber/jaeger/model"
+)
+
+// FilterTags returns span tags that should be inserted into cassandra.
+type FilterTags func(span *model.Span) []TagInsertion
```

Comment:
I am not too psyched with the data dependencies here. Filtering is unrelated to building `TagInsertion`
Why not have the filter only depend on the domain model (e.g. Span and Tag for tags, and Span and Log for logs) and return true/false?